add SYSTEM STOP/START FETCHES and SYNC REPLICA description (#11444)
ClickHouse/ClickHouse: docs/en/sql-reference/statements/system.md

SYSTEM START MERGES [[db.]merge_tree_family_table_name]
```

### STOP FETCHES {#query_language-system-stop-fetches}

> Provides possibility to stop background fetches for inserted parts for tables in the ReplicatedMergeTree family:

Author: nothing happens when I ran `SYSTEM STOP FETCHES` as I see ;)

> Provides possibility to stop background fetches for inserted parts for tables in the ReplicatedMergeTree family:

Author: @blinkov could you provide any side effects when STOP FETCHES will run twice and more?

``` sql
SYSTEM STOP FETCHES [[db.]replicated_merge_tree_family_table_name]
```

### SYNC REPLICA {#query_language-system-sync-replica}

> Wait when ReplicatedMergeTree table will synced with other replicas in cluster, will run until receive_timeout if fetches currently disabled for table:  

Reviewer: ```suggestion
Wait until a `ReplicatedMergeTree` table will be synced with other replicas in a cluster. Will run until `receive_timeout` if fetches currently disabled for the table.
```
